pypeit/scripts/chk_edges.py

```diff
@@ -22,7 +22,10 @@ def parser(options=None):
 
     return parser.parse_args() if options is None else parser.parse_args(options)
 
-
+# TODO: JFH I don't see why we are showing the edges and/or what the purpose of all this if synced not synced is fore.
```

Comment:
Remove this TODO as the Issue captures it and I suspect we'll want both scripts, i.e. why put this here?

Reply:
I'm fine with leaving the comment.  Personally, I see these TODO comments more than I see the GitHub issues, but it's good to have both.

I commented some on the Issue page.  In terms of this comment, I would move it to the EdgeTrace show method.  There is no sync checking here.  You could change the comment, or just change the script, so that it loads the SlitTraceSet and passes that into the EdgeTraceSet show method.

In the EdgeTraceSet show method, the main reason for the `is_synced` checks are for when the show method is called before the traces are synced, which can happen often if you're showing the tracing steps when running `pypeit_trace_edges`.  It's not standard that an EdgeTraceSet file would be written before the slits are synced, but it is possible.  So even here we shouldn't necessarily assume that the EdgeTraceSet is synced.